.npmignore

```diff
@@ -0,0 +1,7 @@
+/assets
+/blueprints/.jshintrc
+/dev
+/docs
+/tests/!(helpers)
+/tmp
+/*.*
```

Comment:
Ah so this doesn't include package.json right? That one is essentially white-listed.